src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const entries = sourceString
+    .split(';')
+    .map((entry) => entry.trim())
+    .filter((entry) => entry);
```

Comment:
The filter function removes empty entries, which is a good practice to avoid processing unnecessary data.